chrissi/index.js

```diff
@@ -0,0 +1,60 @@
+//0
+
+let snackServings = 4;
+let guests = 16;
+
+console.log(snackServings, guests);
+
+// 1 ADD
+console.log(snackServings + guests);
+
+// 2 SUB
+console.log(snackServings - guests);
```

Comment:
You forgot to subtract snackServings from guests!
all you need is to type  console.log(guests - snackServings);

Reply:
done, i overread it